packages/rs-dpp/src/document/extended_document/mod.rs

```diff
@@ -367,6 +367,7 @@ mod test {
             ("byteArray", Value::Bool(true)),
             ("minItems", Value::U64(32)),
             ("maxItems", Value::U64(32)),
+            ("position", Value::U16(0)),
```

Comment:
Is position 32 or 16?

Reply:
It won't really matter as it will get converted to a u64.

Reply:
It just will be confused for people who will read the code so they will think it's u16